examples/live/run.sh

```diff
@@ -204,7 +204,8 @@ python -m mpi4py `which pycbc_live` \
 --single-duration-threshold 7 \
 --single-reduced-chisq-threshold 2 \
 --single-fit-file single_trigger_fits.hdf \
---verbose
+--verbose \
+--psd-variation
```

Comment:
Please leave verbose as the last option, as often you want to turn it off in examples.